mozilla-central-patches/disable-search-tests.diff

```diff
@@ -12,6 +12,11 @@ diff --git a/browser/components/search/test/browser.ini b/browser/components/sea
  [browser_ddg_behavior.js]
 +skip-if = true # mozilla/activity-stream#2339
  [browser_google.js]
+@@ -31,3 +34,3 @@ skip-if = artifact # bug 1315953
+ [browser_google_codes.js]
+-skip-if = artifact # bug 1315953
++skip-if = true # bug 1315953 / mozilla/activity-stream#2366
```

Comment:
Let's start here although there are 3 other `skip-if = artifact` that might just get uncovered after skipping this first one.